src/Parser.js

```diff
@@ -1062,8 +1063,61 @@ export default class Parser {
                     body: arg,
                     star: star,
                 }, "text"), nucleus);
+        } else if (text === "$") {
+            return newDollar(nucleus);
+        }
+        // At this point, we should have a symbol, possibly with accents.
+        // If it's in the symbol table or CJK, we're done.
+        if (symbols[this.mode][text]) {
+            this.consume();
+            return newArgument(new ParseNode(symbols[this.mode][text].group,
+                text, this.mode, nucleus), nucleus);
+        } else if (cjkRegex.test(text)) {
+            this.consume();
+            return newArgument(
+                new ParseNode("textord", text, this.mode, nucleus));
+        }
+        // Decompose symbol into base and combining characters
+        // if the combined symbol is not recognized.
+        text = text.normalize('NFD');
+        // Strip off combining characters
+        const match = combiningDiacriticalMarksEndRegex.exec(text);
+        if (match) {
+            text = text.substring(0, match.index);
+        }
+        // Recognize base symbol
+        let symbol = null;
+        if (symbols[this.mode][text]) {
+            symbol = new ParseNode(symbols[this.mode][text].group,
+                            text, this.mode, nucleus);
+        } else if (cjkRegex.test(text)) {
+            symbol = new ParseNode("textord", text, this.mode, nucleus);
         } else {
-            return null;
+            return null;  // EOF, ^, _, {, }, etc.
+        }
+        this.consume();
+        // Transform combining characters into accents
+        if (match) {
+            for (let i = 0; i < match[0].length; i++) {
+                const accent = match[0][i];
```

Comment:
Are there any unicode glyphs that include multiple accents?  If so, we should test that this works correctly.